src/__tests__/cursor-rules-mdc.helpers.ts

```
@@ -0,0 +1,40 @@
+    // Mock data structure
+    id: 'test-id',
+    name: 'test-name',
+    value: 'test-value'
+  };
+};
+
+export const createMockError = (message: string) => {
+  return new Error(message);
+};
+
+export const waitForAsync = (ms: number = 100) => {
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)